docs/PLUGINS.md

```diff
@@ -290,5 +290,8 @@ You also can write a test case to check export from a live instance is well supp
 
         assertEquals(mapping.getScalarValue("expected-property"), "expected-value");
         assertEquals(mapping.getScalarValue("another-property"), "another-value");
+        
+        List<CNode> yourList = mapping.get("a-list-property").asSequence();
+        assertEquals("list size", 5, yourList.size());
```

Comment:
`yourList.get(0).getScalar().getValue()`

`yourList.get(0).asMapping().getScalarValue('key')`

Reply:
actually I'd like for `CNode` to have the same shortcuts `.get(0).getScalarValue()` and `.get(0).getScalarValue('key')` which handles the scalar and mapping conversion